### PR TITLE
bugherd task #624

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/reference/smartrest-two-bundle/mqtt-static-templates.md [hunk 1]
```diff
@@ -215,9 +215,9 @@ Create a new software service for given device.
 
 |Position|Parameter  |Mandatory|Type  |
 |:-------|:----------|:--------|:-----|
-|1|service name|YES|String|
 |2|service unique external id|YES|String|
 |3|service type|YES|String|
+|1|service name|YES|String|
```
Comment (BeateRixen) [inline content/reference/smartrest-two-bundle/mqtt-static-templates.md:220]: Is it correct that position 1 now is no longer at the top?

Comment (rssag) [reply content/reference/smartrest-two-bundle/mqtt-static-templates.md:220]: oh right. no it isn't

Comment (rssag) [reply content/reference/smartrest-two-bundle/mqtt-static-templates.md]: fixed

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Comment by rssag** [comment]
fixed in develop, 10.15.0 and 10.14.0.

**Pull request merged**